src/nginxconfig/templates/setup.vue

```diff
@@ -163,9 +162,10 @@ THE SOFTWARE.
                 });
 
                 clipboard.on('success', e => {
-                    elm.textContent = 'Copied';
+                    const originalTextCopy = elm.textContent;
+                    elm.textContent = this.$tm('templates.app.copied');
```

Comment:
```suggestion
                    elm.textContent = this.$t('templates.app.copied');
```
I think we just want to use `t` here?